tests/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    'extends': [
+        '../.eslintrc.js',
+        '@jitsi/eslint-config/jsdoc',
+        '@jitsi/eslint-config/react'
```

Comment:
Do we need this one?

Reply:
Do you want to keep the jsdoc one? I was fixing the docs because of it. Keeps code consistent, but I'm not sure it is needed for tests. WDYT?

Reply:
I meant the react one but yeah, we can remove both